kreuzliste/example.yml

```diff
@@ -58,6 +58,24 @@ quotes:
   - "`My hovercraft is full of eels'"
   - "`I will not buy this record; it is scratched'"
   - "`This parrot is no more! It has ceased to be! It's expired and gone to meet its maker!'"
+  - \glqq Das überhandnehmende Maschinenwesen quält und ängstigt mich, es wälzt sich heran wie ein Gewitter, langsam, langsam; \linebreak aber es hat seine Richtung genommen, es wird kommen und treffen.\grqq\ --- Goethe
+  - \glqq Wir sind, was wir wiederholt tun. Exzellenz ist also keine Handlung, sondern eine Gewohnheit.\grqq\ --- Aristoteles
+  - \glqq Das Geheimnis der Freude an der Arbeit ist in einem Wort enthalten - Exzellenz. \linebreak Zu wissen, wie man etwas gut macht, bedeutet, es zu genießen.\grqq\ --- Pearl S. Buck
+  - \glqq Wenn die Erfüllung der Erwartungen der anderen dein oberstes Ziel ist, \linebreak hast du deine Einzigartigkeit und damit deine Exzellenz geopfert.\grqq\ --- Hope Solo
+  - \glqq We don't get a chance to do that many things, and everyone should be really excellent. Because this is our life.\grqq\ --- Steve Jobs
```

Comment:
Hier sollen Englische oder US-Englische Anführungszeichen stehen:

```tex
  - ``We don't get a chance to do that many things, and everyone should be really excellent. Because this is our life.'' --- Steve Jobs
```

Reply:
Ah, sorry, hier muss man die String mit `"` schreiben, ansonsten dreht YAML durch:

```tex
  - "``We don't get a chance to do that many things, and everyone should be really excellent. Because this is our life.'' --- Steve Jobs"
 ```

Reply:
Anführungszeichokalypse